<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP_Socket</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="inputMask">
         <string>000.000.000.000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TCP Socket State</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_tcp_socket_state">
        <property name="text">
         <string>Disconnect</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_Time">
        <property name="text">
         <string>Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="GB_Commom">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Commom</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_read_mesh_dev">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Dev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_Generated_id"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ac_find">
         <property name="text">
          <string>Find AC Device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ota_available">
         <property name="text">
          <string>OTA Available</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ready_for_ota">
         <property name="text">
          <string>Ready For OTA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_force_read_all">
         <property name="text">
          <string>Force Read All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start_rssi_scan">
         <property name="text">
          <string>RSSI(AC)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_send_json">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send Json Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Add Devces</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Generated ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Generated">
            <property name="maxLength">
             <number>22</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Onboard Mac</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_onboard_mac">
            <property name="maxLength">
             <number>12</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Report Interval</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Report_interval">
            <property name="maxLength">
             <number>3</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_add_dev">
            <property name="text">
             <string>Add Devices</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_remove_dev">
            <property name="text">
             <string>Remove_devices</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GB_LB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>LB</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="18" column="0">
        <widget class="QPushButton" name="btn_lb_white_brghtness">
         <property name="text">
          <string>White Brghtness</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QPushButton" name="btn_lb_rgb_brightness">
         <property name="text">
          <string>RGB Brightness </string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QPushButton" name="btn_lb_light_on">
         <property name="text">
          <string>Light ON</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_blue">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#5500ff;&quot;&gt;Blue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="lb_hz_brightness">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QSlider" name="lb_hz_red_color">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QPushButton" name="btn_lb_rgb_light_setting">
         <property name="text">
          <string>RGB Setting </string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_green">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Green&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QSlider" name="lb_hz_green_color">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QPushButton" name="btn_lb_light_off">
         <property name="text">
          <string>Light OFF</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Brightness&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QSlider" name="lb_hz_blue_color">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QProgressBar" name="lb_pb_brightness">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_red">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Red&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>lb_hz_blue_color</zorder>
      <zorder>label_10</zorder>
      <zorder>btn_lb_light_on</zorder>
      <zorder>btn_lb_rgb_light_setting</zorder>
      <zorder>btn_lb_rgb_brightness</zorder>
      <zorder>btn_lb_light_off</zorder>
      <zorder>label_red</zorder>
      <zorder>lb_hz_red_color</zorder>
      <zorder>label_green</zorder>
      <zorder>lb_hz_green_color</zorder>
      <zorder>label_blue</zorder>
      <zorder>lb_hz_brightness</zorder>
      <zorder>lb_pb_brightness</zorder>
      <zorder>btn_lb_white_brghtness</zorder>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="GB_SP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>SP / Boiler</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_sp_on">
           <property name="text">
            <string>General_ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_sp_off">
           <property name="text">
            <string>General OFF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>SP OPC Control</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="rb_sp_enable">
              <property name="text">
               <string>SP OPC Enable</string>
              </property>
              <property name="shortcut">
               <string/>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_sp_disable">
              <property name="text">
               <string>SP OPC disable</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_SP_OPC">
              <property name="text">
               <string>SP OPC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Send Json Format :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Receive Json Format:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextEdit" name="textEdit_send"/>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="textEdit_rec">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>GB_LB</zorder>
   <zorder>GB_Commom</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
